prefixdb_test.go

```diff
@@ -23,6 +25,28 @@ func mockDBWithStuff(t *testing.T) DB {
 	return db
 }
 
+func taskKey(i, k uint32) []byte {
+
```

Comment:
This could probably more simply be

```go
func taskKey(i, k int) []byte { return []byte(fmt.Sprintf("task-%d-key-%d", i, k)) }
```

We don't really care about compactness here, only uniqueness.

Reply:
Thanks for suggestion! I changed it in newest commit.